Tribler/Test/test_my_channel.py

```diff
@@ -94,7 +111,7 @@ def do_add_torrent(torrentfilename):
             managefiles = self.managechannel.fileslist
             manager = managefiles.GetManager()
             manager.startDownload(torrentfilename, fixtorrent=True)
-            manager.startDownloadFromUrl(TORRENT_R, fixtorrent=True)
+            manager.startDownloadFromUrl(r'http://localhost:7286/video.torrent', fixtorrent=True)
```

Comment:
This is done so the URLs in the `test_rss.xml` file are correct. We could dynamically replace these ports in `test_rss.xml` when running the test?